frontend/src/diagram/sprotty/Diagram.types.ts

```diff
@@ -11,6 +11,7 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 
+import 'reflect-metadata';
```

Comment:
Do not import that here, that is a very special dependency already imported at a very special location (`index.ts`). If it's required by a test, import it in the test.

Reply:
Done